packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -947,26 +946,21 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
     const activeRectBottom = Math.floor(activeRect.bottom);
     const targetRectBottom = Math.floor(targetRect.bottom);
     const activeRectTop = Math.floor(activeRect.top);
-    // for paging down
-    if (isForward && targetRectTop < activeRectBottom) {
-      if (!this._shouldWrapFocus(this._activeElement as HTMLElement, NO_VERTICAL_WRAP)) {
-        return LARGE_NEGATIVE_DISTANCE_FROM_CENTER;
+    const isValidCandidateOnpagingDown = isForward && targetRectTop > activeRectBottom;
+    const isValidCandidateOnpagingUp = !isForward && targetRectBottom < activeRectTop;
+
+    if (isValidCandidateOnpagingDown || isValidCandidateOnpagingUp) {
+      if (leftAlignment >= targetRect.left && leftAlignment <= targetRect.left + targetRect.width) {
+        return 0;
+      } else {
+        return Math.abs(targetRect.left + targetRect.width / 2 - leftAlignment);
```

Comment:
Look like this returns in every case so if the outer if is true then there doesn't need to be an else after it

Reply:
👍